### PR TITLE
All orders for Trunkrs

### PR DESCRIPTION
**Pull request by fmsthird**
add an option to force all orders  to be shipped through Trunkrs

### CODE REVIEW

File: includes/wc-internal/orders.php [hunk 1]
```diff
@@ -31,7 +31,8 @@ public function createOrder(string $orderId)
         return;
       $trunkrsOrder = new TRUNKRS_WC_Order($orderId);
 
-      if (!$trunkrsOrder->isTrunkrsOrder)
+      if (!$trunkrsOrder->isTrunkrsOrder
```
Comment (fean) [inline includes/wc-internal/orders.php]: Maybe you could change this to an or instead:

So when you either have enabled the all orders for trunkrs or it is a Trunkrs order we will announce it. 🙂

File: includes/wc-internal/trunkrs-order.php [hunk 2]
```diff
@@ -54,7 +54,8 @@ private function init($withMeta)
 
       foreach ($shippingItem as $item) {
         $shippingMethodId = $item->get_method_id();
-        if ($shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN) {
+        if ($shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN
```
Comment (fean) [inline includes/wc-internal/trunkrs-order.php]: Also goes for this one!

**Review by fean — CHANGES_REQUESTED** [review]
Maybe you could also take care of the other processes. Those take care of automatic cancellation and things like reviving the shipment when it is re-instated after the cancellation.

Also I believe there is a check in the Admin part that determines whether a order is a Trunkrs order and then shows the Trunkrs tools with like the Trunkrs number and label button on it.

File: includes/wc-internal/trunkrs-order.php (hunk 2 above)
Comment (fmsthird) [reply includes/wc-internal/trunkrs-order.php]: `$shippingMethodId !== TRUNKRS_WC_Bootstrapper::DOMAIN` my understanding about this is that when shipping method is not equal to trunkrs we will skip the processing of order for Trunkrs shipping.

**Comment by fean** [comment]
Looks amazing! 💪 Let's merge this!

**Review by fean — APPROVED** [review]

**Pull request merged**